addons/fastroping/functions/fnc_fastRopeLocalPFH.sqf

```diff
@@ -50,6 +50,18 @@ if (isNull attachedTo _unit) exitWith {
         playSound QGVAR(Thud);
     };
 
+    // Holster weapon if one is being held, then unholster it again
+    private _currentWeapon = currentWeapon _unit;
+    if (_currentWeapon != "") then {
+        [_unit] call EFUNC(weaponselect,putWeaponAway);
+        [{
+            params ["_unit", "_weapon"];
+            // Abort if the unit already selected a different weapon
+            if (currentWeapon _unit != "") exitWith {};
```

Comment:
just in case they are uncon
```suggestion
            if (currentWeapon _unit != "") exitWith {};
            if (!([_unit] call EFUNC(common,isAwake))) exitWith {};
```